File: lender/src/main/java/com/example/domain/Lender.java
package com.example.domain;


import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;


@NoArgsConstructor
@AllArgsConstructor
@ToString
@Getter
@Setter
@Document
public class Lender {
    @Id
    private String emailId;
    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String password;
    private String confirmPassword;
    private Address address=new Address();
    private String pan;
    private String aadhaar;
    private byte[] aadhaarImage=new byte[0];
    private  byte[] panImage=new byte[0];
    private double amountToInvest;
    private double interestRate;
    private int loanDuration;
    private String creditScore;
    private double investedAmount;
    private double availableBalance;
}

File: product-webapp/lendinglane/src/app/model/lender.ts


export type Lender = {
    firstName?: any;
    emailId?: any;
    lastName?: any;
    phoneNumber?:any;
    password?: any;
    confirmPassword?: any;
    pan?: any;
    aadhaar?: any;
    address:{
        address?: string;
        city?: string;
        pin?: string;
        state?: string;
    };
    amountToInvest?: any;
    loanDuration?: any;
    creditScore?: any;
    investedAmount?: any;
    availableBalance?: any;
    interestRate?: any;

    

    
}

export type Address= {
    address?: string;
    city?: string;
    pin?: string;
    state?: string;
  }


